data/human/humanity interludes.txt

```diff
@@ -0,0 +1,192 @@
+			`	"Ah, Captain <last>, I'm glad you made it back safely." He pauses to take a sip from his drink. "So, what do you think about the Broadsider? Do you think it will be an effective tool for the Navy?"`
+			choice
+				`	"Not fast enough to target the drones, but should be good against the larger Sestor ships."`
+					goto fastenough
+				`	"A good start, but not enough to make a difference against the Sestor."`
+					goto goodstart
+				`	"Impressive, should help quite a bit."`
+			`	Andre looks relieved. "I take great comfort in hearing that, <first>. There are many of us in the Navy who have had a lot of sleepless nights trying to figure out how we could have done better, and having better tools is a significant part to improving our future response."`
+				goto wrapup
+			label fastenough
+			`	Andre looks thoughtful. "That's good to know. We probably won't be able to make any changes to the current design, but we can recommend that our fleets primarily use them for long-range engagement against capital ships."`
+				goto wrapup
+			label goodstart
+			`	Andre grimaces a bit at that. "That's what I was afraid we were going to hear." He sighs heavily, then continues. "Well, I'm glad it is progress, but we will have to keep working."`
+			label wrapup
+			`	Almost before Andre can finish speaking he is whisked away by a pack of officious aides that definitely separate you from him and deposit him at a table with several of the VIPs. He catches your eye, nods respectfully, and then turns his attention to the dignitaries. Another researcher approaches you to offer their thanks, and hands you an invoice stating that you have been deposited <payment> in exchange for consulting services to the Republic Navy. They inform you that you are welcome to stay as long as you like, but they suspect that Andre will be tied up in meetings for the foreseeable future.`
+			choice
+				`	"Thanks!"`
+					goto finale
+				`	"When will the Broadsider be available?"`
+			`	The aide consults their notes, then replies. "We expect that they should reach our outfitters within the next week or two, although some ships were whitelisted for early deployment. As such, some of our fleet already has them equipped to help secure humanity against future incursions." The aide looks pleased at this. "Well, I must be off. Thank you again for your assistance." They turn, and vanish into the crowd.`
+			label finale
+			`	The broadsider launch event continues on for several more hours, and you have the opportunity to mix and mingle with a variety of dignitaries. There is also a table of food and drinks available. Eventually you decide that there is nothing more to be gained by sticking around, and return to the spaceport.`
```

Comment:
The line `There is also a table of food and drinks available` feels kind of tacked on and out of place. Maybe you could put this in line 167? It would feel more integrated with that line's description of the conference room.

Reply:
Alright, moved that into the initial description of the scene, as suggested. It is the sort of "set the stage" detail that fits better earlier.